libs/barista-components/alert/BUILD.bazel

```
@@ -16,7 +39,54 @@ filegroup(
     ]),
 )
 
+sass_library(
+    name = "scss_lib",
+    srcs = glob(
+        ["src/**/_*.scss"],
+    ),
```

I would specify the file explicitly
```suggestion
    srcs = ":_alert-theme.scss"
```